src/main/java/com/airlines/model/Airplane.java

```diff
@@ -118,7 +72,63 @@ public String toString() {
                 ", manufactureDate=" + manufactureDate +
                 ", capacity=" + capacity +
                 ", flightRange=" + flightRange +
-                ", crew=" + crew +
+                ", crew=" + crewMember +
```

Comment:
why do you think it should be crewMember? 
I don't  see crewMember in Airplanes database scheme